crm-data-components/src/app/extensions/ActionsExtension.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useState } from "react";
+import {
+  Divider,
+  Text,
+  Flex,
+  hubspot,
+  Form,
+  Select,
+} from "@hubspot/ui-extensions";
+import {
+  CrmPropertyList,
+  CrmCardActions,
+  CrmActionButton,
+} from "@hubspot/ui-extensions/crm";
+
+// Define the extension to be run within the Hubspot CRM
+hubspot.extend(({ context, runServerlessFunction, actions }) => (
+  <Extension
+    context={context}
+    runServerless={runServerlessFunction}
+    sendAlert={actions.addAlert}
+  />
+));
```

Comment:
Probably don't need these if they aren't being used in your extension?

```suggestion
hubspot.extend(({ context }) => (
  <Extension
    context={context}
  />
));
```

Reply:
It looks like all three props are required for `Extension` per its type. Even though they're not used, we may still want to leave them to prevent TS errors